src/balancereader/src/main/resources/application.properties

```diff
@@ -1,12 +1 @@
-server.port = ${PORT}
```

Comment:
Is it really safe to remove all this? Where does the server get the $PORT value from now?

Reply:
The PORT env variable was never used in the Java service directly which is why I removed it from application.properties.